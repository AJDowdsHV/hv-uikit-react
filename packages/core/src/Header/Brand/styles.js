const styles = theme => ({
  root: {
    display: "flex",
    alignItems: "center",
    [theme.breakpoints.down("sm")]: {
      position: "absolute",
      left: "50%",
      transform: "translate(-50%, 0)"
    }
  },
  separator: {
    width: 1,
    height: `${theme.hv.spacing.sm}px`,
    margin: `0 ${theme.hv.spacing.xs}px`,
    backgroundColor: theme.hv.palette.accent.acce1
  }
});

export default styles;
