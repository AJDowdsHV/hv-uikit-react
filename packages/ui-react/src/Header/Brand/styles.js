/**
 * Copyright (c) 2018 Hitachi Vantara Corporation.
 *
 *  The copyright to the computer software herein is the property of
 *  Hitachi Vantara Corporation. The software may be used and/or copied only
 *  with the written permission of Hitachi Vantara Corporation or in accordance
 *  with the terms and conditions stipulated in the agreement/contract
 *  under which the software has been supplied.
 */

const styles = theme => ({
  brandContainer: {
    position: "relative",
    display: "inherit",
    alignItems: "center",
    height: "100%",
    marginRight: `${theme.spacing.lg}px`
  },
  iconContainer: {
    display: "inherit",
    maxHeight: "46px"
  },
  companyLogoTextContainer: {
    height: "46px",
    display: "flex",
    alignItems: "center"
  },
  separator: {
    width: `${theme.spacing.sm}px`,
    height: `${theme.spacing.md}px`,
    borderRight: `1px solid ${theme.palette.grey.rainy}`
  },
  separatorPadding: {
    width: `${theme.spacing.sm}px`
  },
  separatorBox: {
    display: "flex"
  },
  productLogoTextContainer: {
    height: "46px",
    display: "flex",
    alignItems: "center"
  },
  paddingTextLeft: { paddingLeft: `${theme.spacing.xs}px` }
});

export default styles;
