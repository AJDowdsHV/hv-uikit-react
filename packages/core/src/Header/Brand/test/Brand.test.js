import React from "react";
import { mount } from "enzyme";

import HvProvider from "../../../Provider";
import Brand from "..";

describe("Brand withStyles", () => {
  let wrapper;

  it("should be able to render", () => {
    wrapper = mount(
      <HvProvider>
        <Brand />
      </HvProvider>
    );

    expect(wrapper.find(Brand)).toMatchSnapshot();
  });
});
