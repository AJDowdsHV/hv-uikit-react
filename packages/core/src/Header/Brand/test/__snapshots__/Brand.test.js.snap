// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Brand withStyles should be able to render 1`] = `
<WithStyles(Brand)>
  <Brand
    classes={
      Object {
        "root": "HvHeaderBrand-root",
        "separator": "HvHeaderBrand-separator",
      }
    }
  >
    <div
      className="HvHeaderBrand-root"
    />
  </Brand>
</WithStyles(Brand)>
`;
